pkgs/applications/window-managers/windowchef/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, libxcb, libXrandr
+, xcbutil, xcbutilkeysyms, xcbutilwm, xcbproto
+}:
+
+stdenv.mkDerivation rec {
+  pname = "windowchef";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner  = "tudurom";
+    repo   = "windowchef";
+    rev    = "v${version}";
+    sha256 = "02fvb8fxnkpzb0vpbsl6rf7ssdrvw6mlm43qvl2sxq7zb88zdw96";
+  };
+
+  buildInputs = [ libxcb libXrandr xcbutil xcbutilkeysyms xcbutilwm xcbproto];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "A stacking window manager that cooks windows with orders from the Waitron";
+    homepage = "https://github.com/tudurom/windowchef";
+    maintainers = with maintainers; [ bhougland18 ];
```

Comment:
I think the best course of action is to change this to what is present in the current maintainers list (`bhougland`) and submit a followup PR later, renaming all instances of `bhougland` to `bhougland18`.

Or, you could just let your maintainer handle be `bhougland` and allow it to be slightly different from your GitHub handle.

Either way works.

Reply:
I will just leave it as bhougland, I use that username many places.

Reply:
Should I do a separate commit for the service module similar to https://github.com/NixOS/nixpkgs/blob/release-19.09/nixos/modules/services/x11/window-managers/spectrwm.nix

Reply:
If you want to add it a service for it, it should definitely be a separate commit, IMO.

Reply:
sounds good.